### PR TITLE
feat(Pictograms): june pictogram additions

### PR DESCRIPTION
**Pull request by dudley-ibm**
Closes `#96, 98, 122, 192`

#### Changelog

**New**
Issue `#96`– Security updated artwork
advocate--mask	(People)
user--mask	(People)
construction-worker	(Industry)
construction-worker--mask	(Industry)

Issue `#98` – Quantum
quantum	(Data analytics)

Issue `#122` – curve estimation models
curve--linear	(Data analytics)
curve--inverse	(Data analytics)
curve--logarithmic	(Data analytics)
curve--quadratic	(Data analytics)
curve--cubic	(Data analytics)
curve--power	(Data analytics)
curve--exponential	(Data analytics)
curve--logistic	(Data analytics)

Issue `#192` – watsonx pictogams
watsonx	(Watson)
watsonx--ai	(Watson)
watsonx--data	(Watson)
watsonx--governance	(Watson)

#### Testing / Reviewing
These show up on Carbon and the IBM Design Language sites

### CODE REVIEW

**Comment by dudley-ibm** [comment]
I have read the DCO document and I hereby sign the DCO.

File: packages/pictograms/pictograms.yml [hunk 1]
```diff
@@ -2843,32 +2816,30 @@
...
+    - Hamburg Philharmone
     - Hamburg
     - Germany
-    - Elbphilharmonie
+    - Philharmone
     - city
     - cities
 - name: handicap
-  friendly_name: Disabled user
+  friendly_name: Handicap
```
Comment (tay1orjones) [inline packages/pictograms/pictograms.yml]: I think we should avoid using this term. It's outdated and generally frowned upon. Unicode calls these symbols "Wheelchair symbol", though it's technically moreso a riff of the [International Symbol of Access](https://en.wikipedia.org/wiki/International_Symbol_of_Access). That would be too cumbersome of a name though imo. The previous name `Disabled user` feels much more appropriate

**Review by tay1orjones — COMMENTED** [review]
I pulled this down and regenerated the metadata with `node tasks/scaffold.js`, but after doing so the pictograms package still fails to build.

File: packages/pictograms/pictograms.yml (hunk 1 above)
Comment (tw15egan) [reply packages/pictograms/pictograms.yml]: Inadvertently pulled in older code

**Review by tay1orjones — APPROVED** [review]
Thanks for helping sort that out @tw15egan! 🙏

**Comment by dudley-ibm** [comment]
> Thanks for helping sort that out @tw15egan! 🙏

Yes! With my connectivity issues last night I may have copied the wrong .yml file. Thank you to @andreancardona too for getting me connected last night!

**Review by tw15egan — APPROVED** [review]

**Review by laurenmrice — APPROVED** [review]

**Comment by stephharrington1** [comment]
@andreancardona @tw15egan 

Hi folks! Do you have an update on when we can expect the watson pictograms to be available? The offerings will be available for public sales this week on 7/7 so we want to be sure they are available for the business to use. Thank you all appreciate your work on this!

cc: @dudley-ibm

**Comment by mjabbink** [comment]
@tw15egan @sstrubberg These are high-priority pictograms needed for communications and various design executions.

**Comment by tw15egan** [comment]
These should be included in `v1.33.0` that will be released this week. Our release candidate `v1.33.0-rc.0` was just pushed out, and we do stable releases 2 days afterwards.

**Comment by dudley-ibm** [comment]
@tw15egan the global announcement of watsonx is this morning and the watsonx pictograms in this pull request from issue 192 are still not live on carbon nor on IBM Design Language. Did these not push out yesterday?

**Comment by tw15egan** [comment]
@dudley-ibm I just merged https://github.com/carbon-design-system/design-language-website/pull/1229 for the design system website, and the other one is up here: https://github.com/carbon-design-system/carbon-website/pull/3629

**Comment by tw15egan** [comment]
@stephharrington1 @dudley-ibm @mjabbink Both websites have been updated to the latest version of Carbon which contains these pictograms 👍🏻 

https://www.ibm.com/design/language/iconography/pictograms/library
https://carbondesignsystem.com/guidelines/pictograms/library/

**Comment by dudley-ibm** [comment]
Awesome. Thank you @tw15egan! They indeed show up now when I search.

**Comment by mjabbink** [comment]
Nice. These are nice. Thanks, @tw15egan — One not, the code names are showing up instead of friendly names.

**Comment by tw15egan** [comment]
@mjabbink friendly names are usually added at a later date by @LMapes 👍🏻

**Comment by dudley-ibm** [comment]
> @mjabbink friendly names are usually added at a later date by @LMapes 👍🏻

@tw15egan actually, Lori usually only adds aliases. It's my input that includes both code names and friendly names, but it appears that the code name ended up in both locations in the pictogram.yml file for those watsonx pictograms. Should I replace the file or is this something that you can update on your end?

**Comment by mjabbink** [comment]
@dudley-ibm I’m referring to the library. 

i.e.

Screenshot 2023-07-12 at 4 21 34 PM
Screenshot 2023-07-12 at 4 21 37 PM

**Comment by dudley-ibm** [comment]
@mjabbink I know you are. I tried to explain above is that the error is in the pictogram.yml file. I've already updated that yml file and will get help from @tw15egan to get it replaced sooner than later.

**Comment by mjabbink** [comment]
@dudley-ibm i was responding to you saying Lori adds it later which made me think you were thinking about something different. I missed the. Second part of you comment. All good. I'm familiar with the process. It's been like 5 years now I've been dealing with icons.

**Comment by tw15egan** [comment]
My bad, I misunderstood! @dudley-ibm submitted a PR to update the friendly names, and we've just merged the changes into the website. Should be live in a few minutes

https://github.com/carbon-design-system/carbon-website/pull/3642

**Pull request merged**